collaborators.md

```diff
@@ -0,0 +1,17 @@
+# Collaborator guide
+
+## Creating releases
+
+Releases are done semi-automatically through GitHub Actions. See the full workflow: [release.yml](.github/workflows/release.yml)
+
+Every first of the month, it attempts to create a release, if there are any unreleased commits on the `main` branch.
+
+To create a release manually, visit the [Release workflow page](https://github.com/sindresorhus/refined-github/actions/workflows/release.yml) and click "Run the workflow". This can be done approximately once a week, if worth it, or earlier if there are bugs of medium severity.
```

Comment:
```suggestion
To create a release manually, visit the [Release workflow page](https://github.com/sindresorhus/refined-github/actions/workflows/release.yml) and click “Run the workflow”. This can be done approximately once a week, if worth it, or earlier if there are bugs of medium severity.
```

Gotta have proper quotes.